handler.js

```diff
@@ -129,6 +129,7 @@ async function handleJob(taskId, rcl) {
             pidusage(pid, function (err, stats) {
                 if (err) {
                     console.error(`pidusage error ${err.code} for process ${pid}`);
+                    pidusage.clear();
```

Comment:
I'm wondering if this is the right place to call `clear()`? This error can occur if the job has started multiple processes (e.g. from a shell script) and one of them has finished, but perhaps others are still running. (This is why we monitor the entire pid tree). Perhaps it is sufficient to call `clear()` just before returning `jobExitCode`?

Reply:
That's right - I didn't think about multiple `pidusage`.